woocommerce/woocommerce-android
# [Payment Method Improvements] Tapping Order Complete button should be equivalent tapping Mark as Paid

Closes: #11583



### Description

This PR is to update the change due correctly based on the user input for how much cash was given.

* I added the `unit-tests-exemption` label as this PR is dependent on others and will likely change. Updating the test is ticketed here:
https://github.com/woocommerce/woocommerce-android/issues/11542


### Testing instructions


1. Enable OTHER_PAYMENT_METHODS
1. Go to an order that needs to be paid
1. Select "Cash"
2. Enter a cash value (try again without entering anything)
3. Tap "Mark Order As Complete" button

Expected: 
* Screen should close and return to the order screen (closing the payment method screen as well)
* The order should be marked as paid/completed
* Amount paid should be accurate on the order
